src/OptFrame/Heuristics/EvolutionaryAlgorithms/PSO.hpp

```diff
@@ -148,6 +148,147 @@ class PSO : public SingleObjSearch<XES, XES2, XSH2>
    }
 
    SearchStatus search(const StopCriteria<XEv>& stopCriteria) override
+   
```

Comment:
It got in the middle of the other file, but ok. It's late now.. we fix tomorrow :)